src/manage_nsfs/health.js

```diff
@@ -480,6 +482,45 @@ class NSFSHealth {
         }
     }
 
+    /**
+     * _get_blocked_hosts_status checks if there are any hosts blocked for updating the config directory
+     * 1. when host's config_dir_version older than 5.18.0 and system's config_dit_version is 5.18.0 and higher it means it's not blocked 
+     * because the source code won't include _throw_if_config_dir_locked() but it's still can cause invalid config files, therefore including it in the blocked list
+     * 2. when system's config_dir_version older than 5.18.0 and hosts's config_dir_version is 5.18.0 and higher
+     * it means updated to the config directory from this host are blocked
+     * @param {Object} system_data 
+     * @returns {Object}
+     */
+    _get_blocked_hosts_status(config_fs, system_data) {
+        const system_config_dir_version = system_data.config_directory?.config_dir_version;
+        const hosts_data = _.omit(system_data, 'config_directory');
```

Comment:
I would move this line to a helper function in case you expend the `system_data` and would need to omit more properties, so we would need to fix it in one place only.